test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -473,4 +473,52 @@ object Assertion {
         case t: Throwable => assertion(t)
       }
     }
+
+  final def isEmptyString: Assertion[String] = Assertion.assertion(s"isEmpty")() { actual =>
```

Comment:
I would make the name of the assertion match the method name (i.e. both "isEmptyString"). Also, you don't need the string interpolation here.